src/screens/expandedAssetSheet/components/sections/AboutSection.tsx

```diff
@@ -94,17 +94,17 @@ function RowButton({ highlighted, icon, iconName, title, url, value }: RowButton
 function truncate(text: string) {
```

Comment:
Wouldn't it be simpler to return early if `text` is falsy? And allow falsy values for `text`, if the string type is being violated somewhere.